3_run_rsync/vars/defaults.yml

```diff
@@ -10,5 +10,8 @@ successful_pvcs_dir: ../output/
 transfer_pod_image: "quay.io/konveyor/stunnel-ssh-transfer:latest"
 mig_dest_ssh_user: root # note: image above has been designed to work with login as root
 
-stunnel_cert_filepath: "./scripts/tls.crt"
-stunnel_key_filepath: "./scripts/tls.key"
+stunnel_cert_filepath: "./files/tls.crt"
+stunnel_key_filepath: "./files/tls.key"
+
+rsyncd_port:  22
+debug_password: true
```

Comment:
Is `debug_password` required?

Reply:
yes, for debugging our envs now, can default this to false or remove right before merging to master